src/python/CRABInterface/Regexps.py

```diff
@@ -77,7 +77,7 @@
 RX_SUBRES_TASK = re.compile(r"^allinfo|allusers|summary|search|taskbystatus|getpublishurl|addwarning|addwebdir|addoutputdatasets|webdir|counttasksbystatus|lastfailures|updateschedd|updatepublicationtime$")
 
 #worker workflow
-RX_WORKER_NAME = re.compile(r"^[A-Za-z0-9\-\._]{1,100}$")
+RX_WORKER_NAME = re.compile(r"^[A-Za-z0-9\-\._%]{1,100}$")
```

Comment:
Actually this is needed because I am using unpercent as wildcard for the asoworker name (which is not know by the PJ, and I don't want to do an additional query to figure it out)